snapshot/src/main/java/io/zeebe/snapshots/broker/impl/FileBasedSnapshotMetadata.java

```diff
@@ -122,34 +122,37 @@ public String getSnapshotIdAsString() {
 
   @Override
   public int hashCode() {
-    return Objects.hash(getIndex(), getTerm(), getTimestamp());
+    return Objects.hash(index, term, processedPosition, exporterPosition);
```

Comment:
You changed the `hashCode()` and `equals()` implementation. 
Why did you change it?
Does this change any behavior?

Reply:
This should have been updated when new fields were added to the class, but it was not.  It won't change any behavior, but it is expected to include processedPosition and exporterPosition when comaparing two snapshots. We did not find out this error, because we are using `SnapshotId::compareTo` to compare them.